### PR TITLE
feat: [CI-5615]: reroute to Test page if execution has failing tests

### PR DESCRIPTION
**Pull request by shaurya-harness**
### Summary

Land onto Test page by default if tests are failing instead of Pipeline in Console view

#### Screenshots

https://user-images.githubusercontent.com/113332561/195318093-40b2e87f-4349-4548-8f52-056de1a47a98.mov

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by shaurya-harness** [comment]
retrigger titlecheck

**Comment by shaurya-harness** [comment]
trigger featurenamecheck

**Comment by shaurya-harness** [comment]
retrigger jest

File: src/modules/10-common/featureFlags.ts [hunk 1]
```diff
@@ -104,5 +104,6 @@ export enum FeatureFlag {
   PL_ENABLE_GOOGLE_SECRET_MANAGER_IN_NG = 'PL_ENABLE_GOOGLE_SECRET_MANAGER_IN_NG',
   CI_DOCKER_INFRASTRUCTURE = 'CI_DOCKER_INFRASTRUCTURE',
   ENABLE_VERIFY_STEP_LONG_DURATION = 'ENABLE_VERIFY_STEP_LONG_DURATION',
-  NEW_LEFT_NAVBAR_SETTINGS = 'NEW_LEFT_NAVBAR_SETTINGS'
```
Comment (udayvunnam) [inline src/modules/10-common/featureFlags.ts:107]: Is this intended? Didn't seem to be used anywhere

**Review by udayvunnam — APPROVED** [review]

**Review by vardanbansal-harness — APPROVED** [review]

**Pull request merged**